usaspending_api/api_contracts/contracts/v2/disaster/agency/spending.md

```diff
@@ -115,7 +115,7 @@ Returns spending details of Agencies receiving supplemental funding budgetary re
 + `code` (required, string)
 + `description` (required, string)
 + `children` (optional, array[Result], fixed-type)
-+ `count` (required, number)
++ `award_count` (required, number)
```

Comment:
Shouldn't `award_count` be nullable for File B results aka "total" spending_type?

Reply:
Actually it isn't present in the response for those situations, it is?

Reply:
I was following our pattern of "include key with null" in this PR, so it's always present, but should be marked nullable.

Reply:
Gotcha. Tracking now.

For the example JSON responses w/ `total_budgetary_resources`: `award_count` should be null